bundled/tbb41_20130401oss/src/tbb/dynamic_link.cpp

```diff
@@ -419,6 +419,7 @@ OPEN_INTERNAL_NAMESPACE
     #endif /* _WIN32 */
 
     static dynamic_link_handle global_symbols_link( const char* library, const dynamic_link_descriptor descriptors[], size_t required ) {
+      (void)library; // Suppress an unused variable warning warning with clang
```

Comment:
`warning warning` -> `warning`

Reply:
how on earth did I manage to write that?